### PR TITLE
Added new Source Types PromptPay and PayNow

### PR DESCRIPTION
**Pull request by tobias90**
Added new source types:
- PromptPay
- PayNow

### CODE REVIEW

**Review by igroomgrim — APPROVED** [review]

File: OmiseSwift/API Models/Source/SourceType.swift [hunk 1]
```diff
@@ -55,6 +57,8 @@ public enum InternetBanking: RawRepresentable, Equatable, Hashable {
 public enum SourceType: Codable, Equatable, Hashable {
     case internetBanking(InternetBanking)
     case alipay
+    case promptpay
+    case paynow
```
Comment (nuxzero) [inline OmiseSwift/API Models/Source/SourceType.swift]: Shall we name these in the camel case? @igroomgrim

Comment (tobias90) [reply OmiseSwift/API Models/Source/SourceType.swift]: Unfortunately right now it's a little bit mixed. TrueMoney is also not named in camel case. But most other source types are named in camel case so it probably makes sense to use it here as well.

**Review by nuxzero — APPROVED** [review]

File: OmiseSwift/API Models/Source/SourceType.swift (hunk 1 above)
Comment (tobias90) [reply OmiseSwift/API Models/Source/SourceType.swift]: Okay I changed the naming to camel case.

**Review by nuxzero — APPROVED** [review]

File: OmiseSwift/API Models/Source/SourceType.swift (hunk 1 above)
Comment (igroomgrim) [reply OmiseSwift/API Models/Source/SourceType.swift]: it's ok for this case 👍

**Review by igroomgrim — APPROVED** [review]

**Pull request merged**